.dockerignore

```diff
@@ -0,0 +1,7 @@
+Dockerfile
+.dockerignore
+node_modules
+npm-debug.log
+README.md
+.next
```

Comment:
```suggestion
.env
.next
```

`.env` is what was causing the issue for me. You can see it's loaded just before build starts.

![Screenshot from 2022-05-11 10-23-52](https://user-images.githubusercontent.com/1779590/167793149-cf07ec3c-bff3-48dd-a6ab-08deed6f0955.png)